lib/fathead/redis_commands/parse.py

```diff
@@ -7,41 +7,58 @@
 url = "http://redis.io"
 output = "output.txt"
 
-f = open(output, "w");
+f = open(output, "w")
 
 tree = lxml.html.parse("download/raw.dat").getroot()
-commands = tree.find_class("command")
+
+"""
+Extract all <li> elements
+First 8 elements are website's meta links, so ignore them
+"""
+elements = tree.findall(".//li")[8:]
```

Comment:
Instead of relying on positioning it might be worthwhile to get the <li> elements within the `container` div.